utils/sample-createspot.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+##  This is sample code that will generally show you how to launch a spot instance on aws and leverage the 
+##  automation built into deepracer-for-cloud to automatically start training
+##  Changes required to work:
+##     Input bucket info
+##     Input security group, iam role, and key-name
+
+## BUCKET is the fundamental input. you need to tell it where in s3 your training will be stored
+## just the bucket name.  no S3:// etc
+
+BUCKET=<########>
+
+
+## Fill these out with your custom information if you want to upload and submit to leaderboard.  not required to run
+DR_UPLOAD_S3_BUCKET=########
+DR_UPLOAD_S3_PREFIX=########
+
+## set the instance type you want to launch
+INSTANCE_TYPE=c5.2xlarge
+
+## if you want to modify additional variables from the default, add them here, then add them to section further below called replace static paramamters.  I've only done World name for now
+WORLD_NAME=FS_June2020
+
+## modify this if you want additional robomaker workers
+DR_WORKERS=1
+
+
+## select which images you want to use.  these will be used later for a docker pull
+DR_SAGEMAKER_IMAGE=cpu-avx-mkl
+DR_ROBOMAKER_IMAGE=2.0.7-cpu-avx2
```

Comment:
Worth storing this without the version number to be future-proof.

Reply:
agreed.  will modify